### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by anatolii-zarytskyi**

### CODE REVIEW

File: src/main/resources/hibernate.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app?serverTimezone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">101351013</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (freedmanMain) [inline src/main/resources/hibernate.cfg.xml:13]: Add an empty line to the end of the file.

File: src/main/java/mate/academy/util/HibernateUtil.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```
Comment (freedmanMain) [inline src/main/java/mate/academy/util/HibernateUtil.java:6]: Add an empty private constructor in this class. 
UPD: This is written in the checklist.

File: pom.xml [hunk 3]
```diff
@@ -20,8 +24,41 @@
             <artifactId>junit</artifactId>
```
Comment (freedmanMain) [inline pom.xml:24]: If you are not using unit tests in your application, remove this dependency.

**Review by freedmanMain — CHANGES_REQUESTED** [review]

**Review by freedmanMain — APPROVED** [review]

File: src/main/java/mate/academy/util/HibernateUtil.java (hunk 2 above)
Comment (RomanDubovskyi) [reply src/main/java/mate/academy/util/HibernateUtil.java:6]: > Add an empty private constructor in this class.
> UPD: This is written in the checklist.

\+

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/util/HibernateUtil.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    private static final SessionFactory instance = initSessionFactory();
+
```
Comment (RomanDubovskyi) [inline src/main/java/mate/academy/util/HibernateUtil.java:8]: and here private

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request closed without merging**